cilium/CFP-22215-mutual-auth-for-service-mesh.md

```diff
@@ -193,7 +295,7 @@ For that lifetime, traffic between Identity A and Identity B will flow without f
     * Per-node SPIRE Agent must be installed in the cluster and configured correctly. The SPIRE agent talks to the SPIRE server over the network, but all other communication with the Cilium Agent is via domain sockets shared on the host's filesystem.
     * Cilium Agent SPIFFE Identity must be created as per SPIRE Agent config.
     * Cilium Operator SPIFFE Identity must be created as per SPIRE Server config.
-    * (This document will be updated after [#23806](https://github.com/cilium/cilium/issues/23806) is done.)
+    * Cilium Operator SPIFFE Identity is granted privileges to manage SPIRE Server entries across the network, so that it can create and delete SPIRE entries for a SPIFFE ID that will cause SVIDs (and the keypairs) to be distributed via the SPIRE system.
```

Comment:
The wording here was a bit confusing. Is this communicating the same information?

```suggestion
    * Cilium Operator SPIFFE Identity is granted privileges to manage SPIRE Server entries across the network. This enables the corresponding SVIDs (and keypairs) to be distributed via the SPIRE system.
```

Reply:
That's much clearer, thanks.